CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ release.
 
 ## [Unreleased]
 
-## [1.0.0] - 2023-11-15 
```

Comment:
Should we update the date on the header too?

Reply:
Yes, I also want to double check we didn't miss anything in the changelog